otx/algorithms/anomaly/tasks/nncf.py

```diff
@@ -144,9 +144,17 @@ def load_model(self, otx_model: Optional[ModelEntity]) -> AnomalyModule:
                     pl_modules[key] = model_data["model"][key]
             model_data["model"] = nncf_modules
 
+            dataloader = None
```

Comment:
I think we can add a type hint for this line

Reply:
Done!